src/App.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-    <div id="main-component">
+    <div
+        id="main-component"
+        :class="{ outlines: showOutlines }"
+        @keydown="setOutlines(true)"
+        @pointerdown="setOutlines(false)"
```

Comment:
Good idea!
I was wondering how we could get rid of this and still enable keyboard navigation.

Reply:
Yeah. Simple but effective 😎 
It can be a bit weird when you click into an input and the outline appears when you start typing. But all in all it's great.

Here's the original implementation used by the style guide:
https://github.com/lindsayevans/outline.js